typescript/cli/src/config/ism.ts

```diff
@@ -71,8 +71,6 @@ const ISM_TYPE_DESCRIPTIONS: Record<IsmType, string> = {
   [IsmType.TRUSTED_RELAYER]: 'Deliver messages from an authorized address',
   [IsmType.TEST_ISM]:
     'ISM where you can deliver messages without any validation (WARNING: only for testing, do not use in production)',
-  [IsmType.OP_STACK]: '',
-  [IsmType.PAUSABLE]: '',
```

Comment:
isn't this just removing the description? the select is generated below in `createIsmConfig`, or am i missing something?

Reply:
maybe something like this?
```typescript
export async function createIsmConfig(
  context: CommandContext,
): Promise<IsmConfig> {
  const moduleType = await select({
    message: 'Select ISM type',
    choices: Object.entries(ISM_TYPE_DESCRIPTIONS).map(
      ([value, description]) => ({
        value,
        description,
      }),
    ),
    pageSize: 10,
  });
...
}
```

Reply:
`IsmType.OP_STACK` and `IsmType.PAUSABLE` are not handled by the if-else tree (another +1 to switch-cases :p ), so they `throw new Error(`Invalid ISM type: ${moduleType}`);` which we decided is poor UX. We do not want them present as options in the selection at all